blog_post.md

```diff
@@ -190,54 +190,55 @@ We use the following hyperparameter settings in all our experiments:
 - discount factor x
       - we try to stick to the original paper as much as possible -->
 
-## Results and Discussion
-
-After identifying the best hyper-parameters for each of our environments, we
-looked at their divergence in four setups: with all tricks disabled, with
-memory sampling, having a target network, and by having both tricks enabled.
-Our results for MountainCar and Acrobot are shown below
-
-![MountainCar divergence]({{page.img_dir}}violinplot_q_divergence_MountainCar-v0_0.99.png)
-![Acrobot divergence]({{page.img_dir}}violinplot_q_divergence_Acrobot-v0_0.99.png)
-
-On the y-axis we see the $$\max |Q|$$ on a logarithmic scale.
-The dashed line represents our theoretical maximal $$|Q|$$, above which we deem
-the network to have diverged.
-We see that with these environments, memory sampling has a limited effect on
-slowing down divergence, whereas employing a separate target network is more
-effective.
-We also observe that when having both tricks,the network's divergence is slowed
-down even more.
-
-Based on all our experiments we can say that divergence with DQN is
-unavoidable, however it can be slowed down, allowing the network to learn more,
-and achieve a higher score.
-We can see this in the following plots, which show us the distribution of
-obtained rewards.
-
-![MountainCar reward]({{page.img_dir}}violinplot_reward_MountainCar-v0_0.99.png)
-![Acrobot reward]({{page.img_dir}}violinplot_reward_Acrobot-v0_0.99.png)
-
-In the case of having both memory sampling and a target network enabled, we
-achieved the highest reward.
-However, the graphs also tell us that although the target network is more
-effective at slowing down divergence, it also negatively influences DQN's
-learning capabilities.
-
-<!--- // TOOD: maybe think of a theoretical explanation of this? -->
-
-Lastly, we look at our results on the CartPole environment, for which we
-explicitly disabled the error clipping.
-Mnih et. al. argued that this improved the stability of the neural network,
-however based on our findings, we can say that it also significantly reduced
-the network's learning capacity (and the reward obtained).
-
-![CartPole divergence]({{page.img_dir}}violinplot_q_divergence_CartPole-v0_0.99.jpeg)
-<!--- <> TODO: add reward image of CartPole. -->
-
-We see that error clipping is instrumental to making the tricks work.
-Without error clipping, the tricks actually hasten the divergence, rather than
-slow it down in our experiments.
+## Results
+
+The results of the best performing hyper-parameters are shown in the below figure.
+The figure displays a scatter plot of DQN's performance in the last 20 episodes of our experiments.
```

Comment:
We also already mention this (last 20 episodes) in the experimental setup.

Reply:
Not really. I checked specifically, and it only says this about the max-|Q| and classifying diverged/not diverged.
Here I specify that each dot is an average/max of the last 20 episodes.